pkg/tfgen/generate_nodejs.go

```diff
@@ -671,14 +670,19 @@ func (g *nodeJSGenerator) emitDocComment(w *tools.GenWriter, comment, docURL, pr
 		}
 	}
 
+	if deprecationMessage != "" {
+		w.Writefmt("%v * ", prefix)
+		w.Writefmt("%v * @deprecated %s */", prefix, deprecationMessage)
+	}
+
```

Comment:
This should be emitted after the docURL below.

Reply:
👍